test/functional/tests/trim/test_trim.py

```diff
@@ -0,0 +1,105 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+import re
+
+from api.cas import casadm
+from core.test_run import TestRun
+from test_utils import os_utils
+from test_utils.size import Size, Unit
+from test_tools.fio.fio import Fio
+from test_tools.blktrace import BlkTrace, BlkTraceMask
+from test_tools.fio.fio_param import ReadWrite, IoEngine
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+
+
+#TODO Add mark for getting discard-supproting devcie
```

Comment:
How about additional check to test if device supports discards? It would prevent test from failing until such mark would be added.

Reply:
Actually this todo can be removed since here cache will be optane or nand disk and it always supports trim (please correct me if I'm wrong)